pkg/framework/framework.go

```diff
@@ -53,6 +53,10 @@ func NewFrameworkWithTimeout(userName string, timeout time.Duration, options ...
 	var k *kubeCl.K8SClient
 	var supplyopts utils.Options
 
+	if options[0].Workspace != "" {
```

Comment:
this will panic in tests that do not specify `options` parameter when calling `NewFramework` function.